federated_aws_cli/login.py

```diff
@@ -54,6 +55,7 @@ def __init__(
         scope="openid",
         token_endpoint="https://auth.mozilla.auth0.com/oauth/token",
         web_console=False,
+        issuer_domain=None
```

Comment:
Trailing comma.  :)

Reply:
Fixed in d12afa1854849ffbb40cf1c2a7e77458ccf7ce54